compiler/plc_driver/src/cli.rs

```diff
@@ -177,6 +179,14 @@ pub struct CompileParameters {
     )]
     pub threads: Option<Threads>,
 
+    #[clap(
+        name = "error-config",
+        long,
+        help = "An diagnostics configuration in JSON format", //TODO: or toml
```

Comment:
can you resolve the todo?


```suggestion
        help = "A diagnostics configuration in JSON format", //TODO: or toml
```

Reply:
I wanted to do this in a different issue, a general toml support for config